content/zh/docs3-building/docs/references/registry/nacos.md

```diff
@@ -0,0 +1,193 @@
+---
+type: docs
+title: "Nacos 注册中心"
+linkTitle: "Nacos"
+weight: 1
+description: "Nacos 注册中心参考手册"
+---
+
+Nacos 是 Dubbo 生态系统中重要的注册中心实现，其中 [`dubbo-registry-nacos`](https://github.com/apache/incubator-dubbo/tree/master/dubbo-registry/dubbo-registry-nacos) 则是 Dubbo 融合 Nacos 注册中心的实现。
```

Comment:
use `https://github.com/apache/dubbo/tree/3.0/dubbo-registry/dubbo-registry-nacos`

Reply:
modify the link address of Dubbo registry Nacos